services/src/main/java/io/druid/cli/validate/DruidJsonValidator.java

```diff
@@ -73,7 +86,23 @@ public void run()
       System.out.printf("File[%s] does not exist.%n", file);
     }
 
-    final ObjectMapper jsonMapper = new DefaultObjectMapper();
+    final Injector injector = GuiceInjectors.makeStartupInjector();
```

Comment:
Basically the thing in the module which binds service and port is required at SOME level by any CLI implementation if it wants to use the druid injection stuff.

The modification in the PR I linked adds these components, and therefore allows using the makeInjector from GuiceRunnable instead of trying to get an injector manually. The hope being that if the injector stuff changes, the functionality in makeInjector will change with it.